src/backend/distributed/sql/udfs/citus_copy_shard_placement/latest.sql

```diff
@@ -1,20 +1,19 @@
+DROP FUNCTION pg_catalog.citus_copy_shard_placement;
 CREATE FUNCTION pg_catalog.citus_copy_shard_placement(
 	shard_id bigint,
 	source_node_name text,
 	source_node_port integer,
 	target_node_name text,
 	target_node_port integer,
-	do_repair bool DEFAULT true,
 	transfer_mode citus.shard_transfer_mode default 'auto')
 RETURNS void
-LANGUAGE C STRICT
+LANGUAGE C
```

Comment:
Removing STRICT seems unintended

Reply:
thanks, I was first experimenting with nullifying do_repair, but in the end did not like the dangling argument very much